akka-cluster/src/test/scala/akka/cluster/ClusterSpec.scala

```diff
@@ -65,12 +64,19 @@ class ClusterSpec extends AkkaSpec(ClusterSpec.config) with ImplicitSender {
 
     "initially become singleton cluster when joining itself and reach convergence" in {
       clusterView.members.size should ===(0)
+      leaderActions()
       cluster.join(selfAddress)
       leaderActions() // Joining -> Up
+      val ccr = cluster.clusterCore ! InternalClusterAction.GetClusterCoreRef
+      println(s"Cluster Core Ref $ccr")
       awaitCond(clusterView.isSingletonCluster)
+      leaderActions()
       clusterView.self.address should ===(selfAddress)
+      leaderActions()
       clusterView.members.map(_.address) should ===(Set(selfAddress))
+      leaderActions()
       awaitAssert(clusterView.status should ===(MemberStatus.Up))
+      leaderActions()
```

Comment:
why change this test, or was it just debugging?

Reply:
Yes, sorry.